wallet-new/src/Cardano/Wallet/Client.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE StrictData #-}
+
+-- | This module defines a client for the Cardano new-wallet.
```

Comment:
Bikeshedding: we might want to simply call it "wallet" dropping the `new` prefix, to legitimate it a bit more 😀

Reply:
Good call :)